lib/lightning_web/live/workflow_live/edit.ex

```diff
@@ -1866,7 +1877,7 @@ defmodule LightningWeb.WorkflowLive.Edit do
     ~H"""
     <div class="relative">
       <div
-        :if={@changeset.changes |> Enum.any?()}
+        :if={@version == "latest" && @changeset.changes |> Enum.any?()}
```

Comment:
Isn't this a double fix? (Like, in a bad way? If you're making it so that the changeset doesn't inaccurately think there are changes, why must you also only show the red dot when version != latest?)

Reply:
Removed it